src/Models/CartRequirement.php

```diff
@@ -111,22 +111,22 @@ public function quantityIsEligible(CartItem $cartItem): array
 
         $nonWhiteGloveProductVariantsInCart = resolve(GoLoadUp::class)->getNonWhiteGloveProductVariantsInCart();
 
-        $firstRequirementsQuantityError = $this->catchQuantityMismatch($cartItem, $nonWhiteGloveProductVariantsInCart, $this->first_set_of_required_shopify_product_ids, );
+        $firstRequirementsQuantityError = $this->catchQuantityMismatch($cartItem, $nonWhiteGloveProductVariantsInCart, $this->first_set_of_required_shopify_product_ids);
 
-        if (! is_null($firstRequirementsQuantityError)) {
+        if (! $firstRequirementsQuantityError[0]) {
```

Comment:
@OniiCoder if `$this->catchQuantityMismatch` returns `null` it would throw an error trying to access `0` index because it's not an array.

Reply:
@BaffourAdu `$this->catchQuantityMismatch` has a default return data which is an array by default, I don't think it would ever return a null after computing. It is either `[false, 'error message']` or `[true, null]`. Do you get?

Reply:
Just the error message which is the 2nd positioned element can be null

Reply:
Noted @OniiCoder , I asked that because of the return type on the method which said `array|null`.

Reply:
It doesn't have the `null` any more just `array`